### DISCUSSION TITLE
Nextjs separate layouts

### DISCUSSION THREAD

**Discussion by MohamedFadelMoumeni** [vercel/next.js#43314] [Help]
Hello guys I have a problem with my nextjs13 project .

My own app directory look like this : 
<img width="399" alt="kkkk" src="https://user-images.githubusercontent.com/56879358/203657891-6bf8171f-f01b-4cbd-9a37-9ce7ce1b6cb5.png">

Tha layout component 
<img width="779" alt="layout" src="https://user-images.githubusercontent.com/56879358/203658028-055d6b25-e880-4e43-8e89-30f763ec73f9.png">

I will create other pages like about / contact  under the rootLayout cause I want to display the header & footer on these pages.
but my login/register pages have different layout . How I can create these pages (login/register) without the rootLayout but I want them under the app directory.

Thanks

**Comment by icyJoseph** [comment]
Hi,

You need to use groups, as documented here, https://beta.nextjs.org/docs/routing/defining-routes#example-creating-multiple-root-layouts. Here's a previous answer, https://github.com/vercel/next.js/discussions/42227#discussioncomment-4017899, that explains how to do this.


```
❯ tree app/
app/
├── (detailed)
│   ├── layout.tsx
│   └── orders
│       └── [id]
│           └── invoice
│               └── page.tsx
└── (root)
    ├── layout.tsx
    ├── orders
    │   └── [id]
    │       └── page.tsx
    └── page.tsx
```

The tree above shows that `/orders/[id]` has one layout (from root), while `orders/[id]/invoice` has a different layout (from detailed).
